README.md

```diff
@@ -102,6 +102,41 @@ if __name__ == '__main__':
     loop.run_until_complete(my_coro_2())
 ```
 
+You may also want to only keep a copy of the context between calls. For example, you have one task that spawns many others and do not want to reflect changes in one task's context into the other tasks.
+To do this use the copying_task_context:
```

Comment:
I think we could reduce the complexity of this example by just having `my_coro_spawner` and another one that changes the context and demonstrating context hasn't changed in `my_coro_spawner`